tests/v2/validation/pipeline/Jenkinsfile.recurring

```diff
@@ -243,6 +243,8 @@ node {
 
                         sh "./yq e '.rancher.clusterName = \"${clusterName}\"' -i ${mainRancherConfigPath}"
 
+                        sh "./yq e '.flags.desiredflags = InstallRancher|Short' -i ${mainRancherConfigPath}"
+
```

Comment:
I think logically this belongs at the end of stage 'deploy clusters' instead of here . i.e. line 203. otherwise lgtm

Reply:
Could we add a comment at the top of this line to emphasize this is to remove the flag, can ignore this if we all agree to add another stage.

This place makes sense to me, we remove that flag _for_ validation tests. We could have another stage after that to make the step clear but this comes with the caveat of qase and config file copies happening in L213. A very similar manipulation happens in L224 for the main configuration. What do you all think?

Is there a specific reason we don't want to put this there? Right after L224, that's changing the main configuration to have a cleanup true similar to this. That way it can update the flags once and for all instead of doing the same operation on the same file in a loop

Reply:
the yq being added is essentially to undo what the code is setting it to, so to me it made sense to do that within the block where the code runs that sets that parameter. I don't feel too strongly, 224 sounds fine too as long as, as you mentioned, its out of this loop.